src/core/redux/selectors/selectCollaborationDatabaseDetails.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * @flow
+ */
+
+import { Map, getIn } from 'immutable';
+import { ValidationUtils } from 'lattice-utils';
+import type { UUID } from 'lattice';
+
+import { COLLABORATIONS, DATABASE_DETAILS } from '../constants';
+
+const { isValidUUID } = ValidationUtils;
+
+export default function selectCollaborationDatabaseDetails(collaborationId :UUID) {
+
+  return (state :Map) :Map => {
+
+    if (isValidUUID(collaborationId)) {
+      return getIn(state, [COLLABORATIONS, DATABASE_DETAILS, collaborationId]) || Map();
+    }
```

Comment:
No change required. `Map()` can be passed as the third parameter to `getIn` function. 
```
return getIn(state, [COLLABORATIONS, DATABASE_DETAILS, collaborationId], Map())
```

Reply:
I've been using `|| Map()` because it guarantees the function will return a value. In theory, it's possible to get `undefined` or `null` when using the `notSetValue` param.